assets/js/theme/auth.js

```diff
@@ -89,15 +89,15 @@ export default class Auth extends PageManager {
     }
 
     registerNewPasswordValidation() {
-        const { password: enterPassword, password_match: matchPassword, invalid_password: invalidPassword } = this.validationDictionary;
+        const { password: enterPassword, password_match: matchPassword } = this.validationDictionary;
         const newPasswordForm = '.new-password-form';
         const newPasswordValidator = nod({
             submit: $(`${newPasswordForm} input[type="submit"]`),
             tap: announceInputErrorMessage,
         });
         const passwordSelector = $(`${newPasswordForm} input[name="password"]`);
         const password2Selector = $(`${newPasswordForm} input[name="password_confirm"]`);
-        const errorTextMessages = createPasswordValidationErrorTextObject(enterPassword, enterPassword, matchPassword, invalidPassword);
+        const errorTextMessages = createPasswordValidationErrorTextObject(enterPassword, enterPassword, matchPassword, this.passwordRequirements.error);
```

Comment:
@jordanarldt, I believe there won’t be so easy way. I see the point of these changes but still we need to think about translations.

We’re supposed to provide a localised versions for all validations like invalid password, email or other input to our merchants. As you can see from the code `this.validationDictionary` is based on `validation_messages` object  from `en.json` (injected via **base.html**, line56). 
Each phrase there (for instance, `invalid_password`) is being translated  to all supported languages and can be found in other lang files. Since you’re removing it and start to use `this.passwordRequirements.error` instead, you should find a way to localise it. 
If you can’t find how it can be implemented via theme then it should be found a responsible team with a newly created ticket.
 
P.S. not sure, but eventually the last option can be more realistic since `this.passwordRequirements.error` text is re-generated based on Merchant’s settings.  

One more direction to think about -  we can use `this.passwordRequirements.error` instead of invalid_password within lang files as a template with some sort of Placeholder(s) for min number of characters (`minlength` field from `this.passwordRequirements` obj) and other parts and replace it/them via js (**assets/js/theme/common/utils/form-utils.js** ) on line 213 with `requirements.minlength` for example.

Reply:
@bc-alexsaiannyi During my testing, I was surprised to find that the **this.passwordRequirements.error** is actually automatically localized already, which means there's no need to add translation keys for this type of error.

![image](https://user-images.githubusercontent.com/19815878/135139374-3fa4f8b8-62ba-4754-892d-56b7ef540e23.png)

Since the passwordRequirements object is automatically generated based on the merchant's password settings, I don't think adding a key in the translation files is a feasible option, which might explain why the message is already localized depending on the store settings.

Reply:
@bc-alexsaiannyi As you mentioned, I do believe that the only other option would be to add the placeholder values in the translation files - but as it looks right now, the current localization that happens automatically on the phrase seems to be good enough, so is adding it to the translation files necessary?

Reply:
@jordanarldt good point! I believe adding a new value to the translation files would be redundant step in this case.